packages/blocto-sdk/src/providers/ethereum.ts

```diff
@@ -583,9 +591,10 @@ export default class EthereumProvider
 
   async handleSendTransaction(payload: EIP1193RequestPayload): Promise<string> {
     this.#checkNetworkMatched();
+    const formatParams = this.#extractParams(payload.params as Array<any>);
```

Comment:
Can we check the format & throw an error if the format is invalid? https://eips.ethereum.org/EIPS/eip-1193#supported-rpc-methods